src/test/java/com/exactpro/epfast/template/assertion/TemplatesComparison.java

```diff
@@ -16,13 +16,19 @@
 
 package com.exactpro.epfast.template.assertion;
 
-import com.exactpro.epfast.template.Reference
-import com.exactpro.epfast.template.simple.Template
-import com.exactpro.epfast.template.simple.Templates
+import com.exactpro.epfast.template.Template;
 
-class TemplatesBuilder internal constructor(val templates: Templates) {
+import java.util.List;
 
-    fun template(name: String, namespace: String = Reference.DEFAULT_NAMESPACE, block: TemplateBuilder.() -> Unit) {
-        templates.templates.add(TemplateBuilder(name, namespace, Template()).apply(block).template)
+import static com.exactpro.epfast.template.assertion.TemplateAssert.assertThat;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TemplatesComparison {
```

Comment:
It's strange you have this class in your Pull request. Probably you added it by mistake during merge. Please check.